### ISSUE TITLE
Error when installing `TypeError: tar.x is not a function`

### ISSUE THREAD

**Issue by webdevian** [crowdin/crowdin-cli#438]
**Describe the bug**
Error when installing JRE `@crowdin/cli@3.7.8`. We didn't have this problem with 3.7.7 (although did have #431)

**To Reproduce**
During Yarn install the below error is thrown

**Environment:**
Installing on Heroku with Node 14.12 and yarn 1.22

**Screenshots**
```
Downloading java runtime environment for version 11
       Failed to install JRE TypeError: tar.x is not a function
           at extractTarGz (/tmp/build_668958cd/node_modules/@crowdin/cli/jdeploy-bundle/jdeploy.js:149:18)
           at /tmp/build_668958cd/node_modules/@crowdin/cli/jdeploy-bundle/jdeploy.js:165:13
/tmp/build_668958cd/scripts/build.js:10
  throw err;
  ^
Missing File
```

**Additional context**
`yarn why tar` return:
```
=> Found "njre#tar@4.4.19"
info This module exists because "@crowdin#cli#njre" depends on it.
info Disk size without dependencies: "232KB"
info Disk size with unique dependencies: "508KB"
info Disk size with transitive dependencies: "612KB"
info Number of shared dependencies: 8
```

**Comment by andrii-bodnar** [comment]
@shannah another issue related to the jDeploy. Update to version 3.1.7 will fix this?

**Comment by shannah** [comment]
@andrii-bodnar Probably.

**Comment by andrii-bodnar** [comment]
🤔

**Comment by webdevian** [comment]
This appears to now be fixed in 3.7.9
